app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/util/CipherViewExtensions.kt

```diff
@@ -88,6 +89,12 @@ fun CipherView.toViewState(
                 zip = identity?.postalCode.orEmpty(),
                 country = identity?.country.orEmpty(),
             )
+
+            CipherType.SSH_KEY -> VaultAddEditState.ViewState.Content.ItemType.SshKey(
+                publicKey = sshKey?.publicKey.orEmpty(),
+                privateKey = sshKey?.privateKey.orEmpty(),
+                fingerprint = sshKey?.fingerprint.orEmpty(),
```

Comment:
👍 `orEmpty()` has to be one of my all time favorite String extensions.

Reply:
Truth